.github/workflows/ci.yml

```diff
@@ -0,0 +1,196 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - auto
+      - try
+    paths-ignore:
+      - 'docs/**'
+      - README.md
```

Comment:
So, we need to run CI all the time when something is pushed to `auto` and `try`, even if that only changes the documentation. If we don't do that, we won't be able to merge those PRs with bors.

Also, now that I think about it, we don't need to run CI on the `master` branch, `auto` and `try` are enough.

Reply:
Jobs frontend and backend are added. Also all `path-ignore` and `master` branch restrctions are now removed :)